src/DIRAC/RequestManagementSystem/private/test/Test_RequestValidator.py

```diff
@@ -75,12 +75,12 @@ def testValidator(self):
         self.assertFalse(ret["OK"])
         self.file.LFN = "/a/b/c"
 
-        # # no ownerDN
-        # force no owner DN because it takes the one of the current user
-        self.request.OwnerDN = ""
```

Comment:
Can you add a backward compatibility test, i.e. a request that has no `Owner` but an `OwnerDN`

Reply:
Is `RequestValidator` running on client side or on server side? or both? If it is running only on Client side then no need add a test here.

Reply:
It's running on both sides

Reply:
The unit test is there, added. There is also a system test in https://github.com/DIRACGrid/DIRAC/blob/integration/tests/System/rms_scripts.sh